evap/staff/tests/test_views.py

```diff
@@ -108,6 +108,79 @@ def test_view_list_sorting(self):
         self.assertLess(position_course1, position_course2)
 
 
+class TestSemesterImportView(ViewTest):
+    url = "/staff/semester/1/import"
+    test_users = ["staff"]
+    filename_valid = os.path.join(settings.BASE_DIR, "staff/fixtures/test_enrollment_data.xls")
+    filename_invalid = os.path.join(settings.BASE_DIR, "staff/fixtures/invalid_enrollment_data.xls")
+
+    @classmethod
+    def setUpTestData(cls):
+        semester = mommy.make(Semester, pk=1)
+        mommy.make(UserProfile, username="staff", groups=[Group.objects.get(name="Staff")])
+        cls.course = mommy.make(Course, pk=1, semester=semester)
```

Comment:
I assumed the "1" in `url = '/staff/semester/1/import'` was this key. The `pk=1` part is all over the file and also not used / needed in some other places except for the link.
Reading through more of the code I juss realized that in the TestSemesterExportView class, there is no pk set for the semester and it's still accessed with .../semester/1/...
I'm new to this stuff, so is my assumption right or not? And if it's right, why is the TestSemesterExportView class working correctly?